frontend/package.json

```diff
@@ -26,8 +25,8 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "prop-types": "^15.6.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "react": "~16.6.0",
+    "react-dom": "~16.6.0",
     "react-hot-loader": "4.0.1",
     "react-transform-catch-errors": "^1.0.2",
     "redbox-react": "^1.5.0"
```

Comment:
PWA 6 contains some major changes which makes it incompatible with earlier versions. So please also add a peer dependency section to the package.json like
```
  "peerDependencies": {
    "@shopgate/pwa-common": "6.x"
  }
```
This will prevent that new extensions with this config can be deployed with older PWA versions.

Reply:
@fkloes please check the latest commit. I've made the requested and discussed changes.